connector/ws/src/lib.rs

```diff
@@ -82,8 +90,30 @@ impl<S: connector::Supervisor> Inner<S> {
             .install_default()
             .unwrap();
 
-        let token_ser_string =
-            serde_json::to_string(&self.config.token).expect("failed to re-serialize token");
+        let token = match self.config.token {
+            WsConnectorConfigToken::TokenFile { ref token_file } => {
```

Comment:
Minor: would it make more sense to match on `&self.config.token` and do away with the `ref`s?

Reply:
Either way works for me.

Reply:
Eh, no need to change it—mostly just curious since I so rarely see `ref`.